docs/dagster-university/pages/dagster-essentials/extra-credit/coding-practice-metadata-taxi-zones-file.md

```diff
@@ -18,7 +18,7 @@ The metadata you built should look similar to the code contained in the **View a
 @asset(
     group_name="raw_files",
 )
-def taxi_zones_file(context):
+def taxi_zones_file():
```

Comment:
D'you think we need to add or mention adding the `MaterializeResult` import? It looks like we add it earlier in the lesson to update the `taxi_trips_file` asset, but curious if you think it's worth it to do.